recipes/zbar/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "without_video": [True, False],
+               "without_imagemagick": [True, False],
+               "without_gtk": [True, False],
+               "without_qt": [True, False],
+               "without_python_bindings": [True, False],
+               "with_x": [True, False],
+               "without_xshm": [True, False],
+               "without_xv": [True, False],
+               "without_jpeg": [True, False],
+               "disable_pthread": [True, False],}
+    default_options = {'shared': False,
+                       'fPIC': True,
+                       'without_video': True,
+                       'without_imagemagick': True,
+                       'without_gtk': True,
+                       'without_qt': True,
+                       'without_python_bindings': True,
+                       'with_x': False,
+                       'without_xshm': False,
+                       'without_xv': False,
+                       'without_jpeg': False,
+                       'disable_pthread': False,
+                       }
+
+    _env_build = None
+    _env_args = []
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_autotools(self):
+        if not self._env_build:
+            self._env_build = AutoToolsBuildEnvironment(self)
+            if self.options.without_video:
+                self._env_args.extend(["--disable-video"])
+            if self.options.without_imagemagick:
+                self._env_args.extend(["--without-imagemagick"])
+            if self.options.without_gtk:
+                self._env_args.extend(["--without-gtk"])
+            if self.options.without_qt:
+                self._env_args.extend(["--without-qt"])
+            if self.options.without_python_bindings:
+                self._env_args.extend(["--without-python"])
+            if self.options.with_x:
+                self._env_args.extend(["--with-x"])
+            if self.options.without_xshm:
+                self._env_args.extend(["--without-xshm"])
+            if self.options.without_xv:
+                self._env_args.extend(["--without-xv"])
+            if self.options.disable_pthread:
+                self._env_args.extend(["--disable-pthread"])
+            if self.options.without_jpeg:
+                self._env_args.extend(["--without-jpeg"])
+            if self.options.shared:
+                self._env_args.extend(["--enable-shared", "--disable-static"])
+            else:
+                self._env_args.extend(["--enable-static", "--disable-shared"])
+        return self._env_build, self._env_args
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Zbar can't be built on Windows")
```

Comment:
I'm not really sure about this, as I said before I have no experience developing on Windows. On the `INSTALL` file in the source code there's a line that says that you can download an installer for Windows but there are no instructions for building it.

Reply:
You should try madebr's approch: https://github.com/conan-io/conan-center-index/blob/0dc94bf7ab7e0e035dd8f91a96be0cb30628316d/recipes/autoconf/all/test_package/conanfile.py#L16

Reply:
Check this recipe: https://github.com/conan-io/conan-center-index/pull/821/files